pkgs/tools/security/kubeclarity/default.nix

```
@@ -17,19 +18,23 @@ buildGoModule rec {
     hash = "sha256-buEahr6lr+C/99ANAgYdexPX76ECW6yGMes8u2GZKh4=";
   };
 
-  vendorHash = "sha256-eAqF0ohZGryRh4u+j/30BObYP23yyrTecPrt+xmn9Sg=";
+  vendorHash = "sha256-JY64fqzNBpo9Jwo8sWsWTVVAO5zzwxwXy0A2bgqJHuU=";
+
+  proxyVendor = true;
 
   nativeBuildInputs = [
     pkg-config
   ];
 
-  buildInputs = [
+  buildInputs = [] ++ (lib.optionals stdenv.isLinux [
```

```suggestion
  buildInputs = lib.optionals stdenv.isLinux [
```

When condition is false, it returns a blank list.

```
nix-repl> lib.optionals false [ 1 ]
[ ]
```